src/python/bindings.cpp

```diff
@@ -504,6 +523,12 @@ PYBIND11_MODULE(pyqmap, m) {
           "n_threads_heuristic", &cs::Configuration::nThreadsHeuristic,
           "Maximum number of threads used for the heuristic optimizer. "
           "Defaults to the number of available threads on the system.")
+      .def_readwrite("gate_set", &cs::Configuration::gateSet,
+                     "Gate Set to be used for the Synthesis. "
+                     "Defaults to {H, S, Sdg, X, Y, Z, CX}.")
```

Comment:
This comment is not really true. `CX` newer really plays into whatever is being set here and there is no way to specify other two-qubit gates. Guess this is mostly a consequence of the inappropriate naming.